site2/docs/performance-pulsar-perf.md

```diff
@@ -109,21 +123,33 @@ The following table lists configuration options available for the `pulsar-perf c
 | acks-delay-millis | Set the acknowledgment grouping delay in milliseconds. | 100 ms |
 | auth-params | Set the authentication parameters, whose format is determined by the implementation of the `configure` method in the authentication plugin class, such as "key1:val1,key2:val2" or "{"key1":"val1","key2":"val2"}". | N/A |
 | auth_plugin | Set the authentication plugin class name. | N/A |
+| auto_ack_chunk_q_full | Configure whether to auto ack for oldest message on queue is full. | false |
```

Comment:
Sorry, this is a grammar mistake in this sentence, can you rewrite it?

Reply:
Okay